### ISSUE TITLE
Simple-unless Reports Same Value for Start and End Location

### ISSUE THREAD

**Issue by judithhinlung** [ember-template-lint/ember-template-lint#2939]
Currently, the 'simple-unless' rule is reporting the same value for start and end locations for certain indentation issues.

```
# examples
  {{#unless this.foo}}
    'Bingo!'
  {{else}}
    'Bogus!'
  {{/unless}}
```

Sample output:
[
  {
    "column": 0,
    "endColumn": 0,
    "endLine": 3,
    "filePath": "layout.hbs",
    "isFixable": true,
    "line": 3,
    "message": "Using an {{else}} block with {{unless}} should be avoided.",
    "rule": "simple-unless",
    "severity": 2,
    "source": "{{else}}",
  }
]

This bug could cause static analysis tools to throw an exception when using Ember-template-lint to scan for issues.
